(Docs) getStaticProps & getServerSideProps TS section (#40607) (#40639)
vercel/next.js: docs/api-reference/data-fetching/get-server-side-props.md

### getServerSideProps with TypeScript

> To specify type of `getServerSideProps` explicitly, you can use `GetServerSideProps` from `next`:

Reviewer: ```suggestion
The type of `getServerSideProps` be specified using `GetServerSideProps` from `next`:
```
